scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -130,11 +121,13 @@
       }
 
       .crm-accordion-body {
-        padding: 15px 0 0 !important;
-
         .section-shown {
           padding: 0;
         }
+
+        .status.no-popup {
+          margin: 0;
```

Comment:
why this custom margin?

Reply:
Parent is already having padding 20px so, we need to remove any default margin to this